application.py

```diff
@@ -33,6 +31,7 @@ def get_service_categories():
     """
     return [
         ("CMIP6", "/cmip6"),
+        ("CMIP6, Downscaled", "/cmip6_downscaled"),
```

Comment:
I agree with this routing: fundamentally a different resource than the coarser cmip6